### ISSUE TITLE
feature logits problem when num_classes is small

### ISSUE THREAD

**Issue by marvis** [microsoft/Cream#3]
Hello,

Thanks for your great work! Your idea is brilliant!

The meta network use feature logits as input to compute the fitness between sub-networks. But when the num classes is small such as 2, the information loss in feature logits could be very severe, which will makes the meta network hard to learn. 

So have you use the feature before last fully connection to learn the meta network? If so, can you share the result.

Best,

**Comment by mapleam** [comment]
Hi,

Thanks for your interest in our work!

For now, our search experiments are performed on ImageNet(Number of classes is 1000). We haven't tried datasets with small number of classes. This could be an interesting problem in our future work - How to select the teacher model with limited class information. 

We have tried the feture map before the last fully connection. However, using feature map brings no significant improvement and increase the training time. So, we just use the last fully connection as the input of the meta matching network.

Best,
Hao.

**Comment by marvis** [comment]
thanks for your reply ~
